com.regnosys.rosetta.lib/pom.xml

```diff
@@ -11,16 +11,25 @@
     <artifactId>com.regnosys.rosetta.lib</artifactId>
     <packaging>eclipse-plugin</packaging>
 
+	<!-- override java versions in lib projects for wider compatibility -->
+	<properties>
+		<!-- release property used by maven compiler and tycho plugins -->
+		<maven.compiler.release>8</maven.compiler.release>
+	</properties>
+
     <build>
         <plugins>
             <plugin>
                 <groupId>org.eclipse.xtend</groupId>
                 <artifactId>xtend-maven-plugin</artifactId>
+                <configuration>
+                	<javaSourceVersion>${maven.compiler.release}</javaSourceVersion>
+                </configuration>
```

Comment:
Actually, I don't think we need this plugin at all in `lib`. We don't (and shouldn't) use xtend in our lib project.

Reply:
I agree we shouldn't, but we do at the moment `com.rosetta.model.lib.expression.MapperMaths.xtend`